src/App.tsx

```diff
@@ -1,173 +1,89 @@
-import { Loader } from './components/Loader';
+import {
+  Navigate, Route, Routes,
+} from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { PeopleTable } from './components/PeopleTable';
+import { PageNav } from './components/PageNav';
+import { ErrorNotification } from './components/ErrorNotification';
 
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+import { Person } from './types/Person';
+import { getPeople } from './api';
+
+export const App = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isPeopleLoaded, setIsPeopleLoaded] = useState(false);
```

Comment:
the variable means the past time, not the process, and we need the loading process and show the loader with it
```suggestion
  const [isPeopleLoading, setIsPeopleLoading] = useState(false);
```